docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsWorkflowEdit.js

```diff
@@ -142,6 +142,12 @@ angular.module('docs').controller('SettingsWorkflowEdit', function($scope, $dial
         actions: [],
         actionType: 'ADD_TAG'
       }];
+    } else if (step.type === "RESUME_REVIEW") {
+      step.transitions = [{
+        name: 'RESUME_REVIEW',
```

Comment:
This needs to be changed to match the [documented format](https://github.com/CMU-313/fall22-hw3-extreme-sharks/blob/7b4b0051083a3485abc437bd48fc013525913482/docs-web/src/main/java/com/sismics/docs/rest/resource/RouteModelResource.java#L89). You can apply the following fix:

```suggestion
        name: 'REVIEWED',
```